app/views/preferences/index.html.erb

```diff
@@ -77,7 +77,7 @@
       <%= f.select :skin_id, @available_skins.collect{|s| [s.title, s.id]} %>
     </dd>
 
-    <dt><%= f.label :time_zone, ts('Your time zone: ') %></dt>
+    <dt><%= f.label :time_zone, ts('Your time zone ') %></dt>
```

Comment:
Can you remove the extra space after "zone", i.e. `<%= f.label :time_zone, ts('Your time zone') %>`?

Also, line 84 needs a space between the `)` and the `%`:  `<%= f.label :preferred_locale, ts('Your locale') %>`

The colons on lines 74 and 88 also need to go. (They're not in the translation strings, so they're easy to overlook.)